ui-tests/tst/software/aws/toolkits/jetbrains/uitests/tests/SQSTest.kt

```diff
@@ -50,16 +50,21 @@ class SQSTest {
     private val createQueueText = "Create Queue..."
     private val deleteQueueText = "Delete Queue..."
     private val purgeQueueText = "Purge Queue..."
+    private val subscribeToSnsText = "Subscribe to SNS topic..."
     private val editQueueAttributesAction = "Edit Queue Attributes..."
     private val editQueueAttributesTitle = "Edit Queue Attributes"
 
     private val queueName = "uitest-${UUID.randomUUID()}"
+    private val snsTopicName = "uitest-${UUID.randomUUID()}"
+    private var snsTopicArn = ""
     private val fifoQueueName = "fifouitest-${UUID.randomUUID()}.fifo"
 
     @Test
     @CoreTest
     fun testSqs() = uiTest {
+        val snsClient = SnsClient.create()
```

Comment:
rather than having this whole thing commented out - can we wrap it with an "assume" or something? I'm worried we're going to forget to uncomment it when we got to turn this on.

Reply:
Talked offline, we can't really do this well today, we will revisit a fix for this issue.